targets/simple_switch/simple_switch.h

```diff
@@ -79,14 +74,17 @@ class SimpleSwitch : public Switch {
   };
 
   static constexpr port_t default_drop_port = 511;
+  static constexpr size_t default_nb_queues_per_port = 1;
 
  private:
   using clock = std::chrono::high_resolution_clock;
 
  public:
   // by default, swapping is off
   explicit SimpleSwitch(bool enable_swap = false,
-                        port_t drop_port = default_drop_port);
+                        port_t drop_port = default_drop_port,
+                        size_t nb_queues_per_port
+                            = default_nb_queues_per_port);
```

Comment:
Would it be possible to have `size_t nb_queues_per_port = default_nb_queues_per_port` on a single line?

Reply:
I see the column of the semicolon is 81 in VSCode, so I put it on two lines. Is it better to put on a single line?

Reply:
Oh, I made a mistake. It is on the 80th column. I will fix this.